tests/usecase/test_db_config_options.py

```diff
@@ -0,0 +1,85 @@
+import os
+from database_config import get_database_config
+
+
+class TestDatabaseConfigOptions:
+    def clean_env(self):
+        """
+        Setting different environment variables for test in the same process
+        can lead to flaky tests.
+        """
+        if "DATABASE_URL" in os.environ.keys():
+            del os.environ["DATABASE_URL"]
+
+        for key in filter(
+            lambda x: x.startswith("MXRUNTIME_Database"),
+            list(os.environ.keys()),
+        ):
+            del os.environ[key]
+
+    def test_no_setup(self):
+        self.clean_env()
+        try:
+            config = get_database_config()
```

Comment:
If I understand correctly, in this case this call is supposed to raise an exception, meaning the `assert config is None` will never be called.
Furthermore, could you use a `with self.assertRaises(RuntimeError)` (used RuntimeError in the example as per @joshp123's comment) context manager to assert the exception was raised?

Reply:
👍
One last thing here: since `config` is not used, you can remove the assignment and just leave the function call.

Reply:
that's done as well